docs/contents/components/typography/heading/index.ja.mdx

```diff
@@ -22,40 +22,28 @@ import { Heading } from "@yamada-ui/react"
 
 ### サイズを変更する
 
-```tsx
-<VStack>
-  <Heading as="h1" size="4xl" isTruncated>
-    ギャルのパンティーおくれーーーっ！！！！！
-  </Heading>
-
-  <Heading as="h2" size="3xl" isTruncated>
-    ギャルのパンティーおくれーーーっ！！！！！
-  </Heading>
-
-  <Heading as="h2" size="2xl" isTruncated>
-    ギャルのパンティーおくれーーーっ！！！！！
-  </Heading>
-
-  <Heading as="h3" size="xl" isTruncated>
-    ギャルのパンティーおくれーーーっ！！！！！
-  </Heading>
-
-  <Heading as="h3" size="lg" isTruncated>
-    ギャルのパンティーおくれーーーっ！！！！！
-  </Heading>
-
-  <Heading as="h4" size="md" isTruncated>
-    ギャルのパンティーおくれーーーっ！！！！！
-  </Heading>
-
-  <Heading as="h5" size="sm" isTruncated>
-    ギャルのパンティーおくれーーーっ！！！！！
-  </Heading>
-
-  <Heading as="h6" size="xs" isTruncated>
-    ギャルのパンティーおくれーーーっ！！！！！
-  </Heading>
-</VStack>
+```tsx functional=true
+const items = [
+  { as: "h1", size: "4xl" },
+  { as: "h2", size: "3xl" },
+  { as: "h2", size: "2xl" },
+  { as: "h3", size: "xl" },
```

Comment:
@illionillion
The `as="h3" size="lg"` specification has been removed - is this intentional?

Reply:
@chida09 
That was my mistake.
Thanks for pointing that out.